src/components/Habit/Habit.js

```diff
@@ -1,24 +1,83 @@
 import React from 'react'
+import IconDelete from '../../components/assets/icons/IconDelete.svg'
 import styled from 'styled-components'
+import imagePathCircle from '../assets/icons/circle.svg'
+import imagePathCheckedCircle from '../assets/icons/checked-circle.svg'
 
-export default function Habit({ name, completed, id, onItemClick }) {
+export default function Habit({
+  category,
+  habits,
+  completed,
+  id,
+  onRemove,
+  onItemClick,
+  color,
+}) {
   return (
-    <HabitItem className="Habit">
-      {name}
-      <input
-        onChange={() => onItemClick(id)}
-        checked={completed}
-        type="checkbox"
-      />
-    </HabitItem>
+    <>
+      <Container style={{ backgroundColor: color }}>
+        <StyledCategory>{category}</StyledCategory>
+        <StyledHabit>
+          <DeleteButton type="button" onClick={() => onRemove(id)}>
+            <img src={IconDelete} alt="Delete habit" />
+          </DeleteButton>
+          {habits}
+          <label>
```

Comment:
Do you need the label here?

Reply:
I do, HiddenCheckbox & StyledCheckbox belong together. If I leave it out, the checkbox is triggered, when I click on the Delete icon. I wonder, if there is a different solution?